docs/configuration-options.md

```diff
@@ -1,33 +1,74 @@
+```kotlin
+// build.gradle.kts
+benchmark {
+    configurations {
+        register("smoke") {
+            // Configure this configuration profile here
+        }
+        // here you can create additional profiles
+    }
+}
+```
+Configuration profiles enable you to execute benchmarks in specific ways.
+`include` and `exclude` options are used to specify the benchmarks to run in this profile.
+By default, all benchmarks are included.
+`kotlinx-benchmark` Gradle plugin creates tasks for running benchmarks in each configuration profile. For example, 
+task `smokeBenchmark` will run benchmarks according to the `"smoke"` configuration profile. See [tasks-overview.md](tasks-overview.md).
+Use the options described below to configure how the benchmarks should be run.
+
+## Core Configuration Options
+
+Be aware that values defined in the build script will override those specified by annotations in the code.
+
+| Option                              | Description                                                                                                                  | Possible Values               | Corresponding Annotation                            |
+| ----------------------------------- |------------------------------------------------------------------------------------------------------------------------------|-------------------------------|-----------------------------------------------------|
+| `iterations`                        | Sets the number of iterations for measurements.                                                                              | Integer                       | @Measurement(iterations: Int, ...)                  |
+| `warmups`                           | Sets the number of iterations for system warming, ensuring accurate measurements.                                            | Integer                       | @Warmup(iterations: Int)                            |
+| `iterationTime`                     | Sets the duration for each iteration, both measurement and warm-up.                                                        | Integer                       | @Measurement(..., time: Int, ...)                   |
+| `iterationTimeUnit`                 | Defines the unit for `iterationTime`.                                                                                        | Time unit, see below          | @Measurement(..., timeUnit: BenchmarkTimeUnit, ...) |
+| `outputTimeUnit`                    | Sets the unit for the results display.                                                                                       | Time unit, see below          | @OutputTimeUnit(value: BenchmarkTimeUnit)           |
+| `mode`                              | Selects "thrpt" for measuring the number of function calls per unit time or "avgt" for measuring the time per function call. | "thrpt", "avgt"               | @BenchmarkMode                                      |
```

Comment:
"Throughput" and "AverageTime" are also possible. They are synonyms for "thrpt" and "avgt" correspondingly. See
https://github.com/Kotlin/kotlinx-benchmark/blob/b5e0a233769a626d2b8318f6389c922ef78df149/runtime/commonMain/src/kotlinx/benchmark/CommonBenchmarkAnnotations.kt#L43

Reply:
Noted. I have added both to the table in the latest commit